components/organisms/UserSettingsPage/user-settings-page.tsx

```diff
@@ -433,19 +434,22 @@ const UserSettingsPage = ({ user }: UserSettingsPageProps) => {
             >
               Update Preferences
             </Button>
-          </div>
+          </fieldset>
           {userInfo && (
             <>
               <div>
                 {!coupon ? (
                   <div className="flex flex-col order-first gap-6 md:order-last">
                     <div className="flex flex-col gap-3">
-                      <label className="text-2xl font-normal">Developer Pack</label>
+                      <label htmlFor="verify-account-button" className="text-2xl font-normal">
+                        Developer Pack
+                      </label>
                       <div className="w-full sm:max-w-80">
                         <Text>Verify your developer pack eligibilty to get an upgrade!</Text>
                       </div>
                     </div>
                     <DeveloperPackForm
+                      id="verify-account-button"
```

Comment:
The `id` prop on all the DeveloperPackForm components is unnecessary as the Button component in the component has text, so no need to connect it to a label.